components/EventItem.tsx

```diff
@@ -1,326 +1,177 @@
+            </View>
+          </Shadow>
         </View>
       </View>
-    </View>
+    </Shadow>
   )
 }
 
-export default React.memo(EventItem)
-
 const styles = StyleSheet.create({
-  secondaryContainerStyle: {
-    backgroundColor: "#f7f7f7",
-    paddingTop: "2%"
-    // paddingVertical: "5%"
-    // borderWidth: 2
-  },
   flexRow: {
     flex: 1,
     flexDirection: "row"
   },
-  spacingTop: {
-    paddingTop: "2%"
+  topRow: {
+    paddingBottom: "4%",
+    alignItems: "center"
   },
-  eventContainerStyle: {
-    paddingBottom: "3%",
-    paddingTop: "1%"
-  },
-  eventTitle: {
+  middleRow: {
     flex: 1,
-    flexDirection: "row",
-    alignSelf: "center",
-    paddingLeft: "1%",
-    fontSize: 18,
-    color: "grey"
+    flexDirection: "column"
   },
-  distance: {
-    flex: 1,
-    flexDirection: "row",
-    alignSelf: "center",
-    textAlign: "right",
-    paddingRight: "4%",
-    fontSize: 18,
-    color: "grey"
+  location: {
+    paddingBottom: "3%"
   },
-  divider: {
-    width: "92%",
-    height: 1,
-    alignSelf: "center"
+  infoText: {
+    textAlignVertical: "center",
+    color: "grey",
+    paddingLeft: "2%"
   },
-  description: {
-    paddingBottom: "3%",
-    paddingTop: "2%"
+  attendingNumber: {
+    fontWeight: "bold",
+    paddingLeft: "5%"
   },
-  iconsBottomRight: {
-    flex: 1,
-    flexDirection: "row",
-    justifyContent: "flex-end"
+  attendingText: {
+    textAlignVertical: "center",
+    fontSize: 14
   },
-  numbersBottomRight: {
+  titleText: {
     textAlignVertical: "center",
-    textAlign: "right",
-    fontSize: 16
+    fontWeight: "bold",
+    fontSize: 22,
+    paddingBottom: "1%"
   },
-  alignIcon: {
-    flexDirection: "row",
-    alignItems: "center",
-    paddingLeft: "2%"
+  name: {
+    textAlignVertical: "center",
+    fontWeight: "bold",
+    fontSize: 15,
+    paddingLeft: "3%"
   },
-  eventDot: {
-    flexDirection: "row",
-    alignItems: "center",
-    paddingTop: "1%",
-    paddingHorizontal: "1.5%"
+  moreButtonStyle: {
+    flex: 1,
+    alignItems: "flex-end"
   },
-  profile: {
+  distance: {
     flexDirection: "row",
-    paddingLeft: "3%",
-    paddingTop: "2%"
+    alignSelf: "center",
+    paddingVertical: "3%",
+    borderRadius: 14,
+    borderWidth: 3
   },
-  maxLimit: {
+  distanceContainer: {
     flexDirection: "row"
   },
-  numbersBottomLeft: {
+  distanceText: {
     textAlignVertical: "center",
-    fontSize: 16
+    color: "white",
+    paddingRight: "3%",
+    paddingLeft: "1%",
+    fontWeight: "bold"
+  },
+  image: {
+    width: 40,
+    height: 40,
+    borderRadius: 24
   },
-  nestedReply: {
+  container: {
     backgroundColor: "white",
-    shadowColor: "#000",
-    shadowOffset: {
-      width: 0,
-      height: 1
-    },
-    shadowOpacity: 0.18,
-    shadowRadius: 1.0,
-    elevation: 1
+    paddingHorizontal: "6%",
+    paddingVertical: "3%",
+    borderRadius: 20
   }
 })
+
+export default React.memo(EventItem)
```

Comment:
I don't think this memo actually does anything since the event props will just be compared referentially (not to mention the callback functions that we'll likely need to add on to this). We'll need some way to check whether or not the props are _actually_ equal for this memo to do anything substantial.

(Chances are that we probably won't need to memo unless we actually have lag in the app)